azure-iot-device/azure/iot/device/common/pipeline/pipeline_nucleus.py

```diff
@@ -4,10 +4,26 @@
 # license information.
 # --------------------------------------------------------------------------
 
+from enum import Enum
+
 
 class PipelineNucleus(object):
     """Contains data and information shared across the pipeline"""
 
     def __init__(self, pipeline_configuration):
         self.pipeline_configuration = pipeline_configuration
-        self.connected = False
```

Comment:
>connected [](http://example.com/codeflow?start=13&length=9)

Did you miss the MQTTPipeline.connected property?

Reply:
No. It points to this new property here.